modules/govuk/manifests/apps/local_links_manager.pp

```diff
@@ -128,6 +128,12 @@
         host_name           => $::fqdn,
         freshness_threshold => (25 * 60 * 60), # 25 hrs (e.g. just over a day)
       }
+
+      @@icinga::passive_check { "local-links-manager-export-links_${::hostname}":
+        service_description => 'Export links to CSV',
```

Comment:
The `service_desc` in the lolotron is ["Export links to CSV from local-links-manager"](https://github.com/alphagov/local-links-manager/blob/5be7553e9f2390bbcbb203ca247b0f88f69ccb60/lib/tasks/export/link_exporter.rake#L8)